ivy/functional/backends/paddle/layers.py

```diff
@@ -494,3 +495,40 @@ def conv_general_transpose(
     if data_format == "channel_last":
         res = res.transpose(0, *range(2, dims + 2), 1)
     return res
+
+
+# @to_native_arrays_and_back
```

Comment:
let's remove this implementation if we're deferring to the compositional implementation for `paddle`